lang/fr-fr.js

```diff
@@ -0,0 +1,47 @@
+/* eslint quotes: 0 */
+
+export default {
+	"components.alert.close": "Fermer l’alerte",
```

Comment:
Not sure how we feel about these special apostrophes... in `fr-CA` we replaced them with normal ones.